core/includes/icon.inc

```diff
@@ -282,7 +282,7 @@ function icon_get_info($icon_name = NULL) {
  *   - attributes: Attributes to be added to the icon itself.
  *
  * @return string
- *   The HTML output.
+ *   The HTML output. An empty string if the icon file is not svg.
```

Comment:
Can we please add another `@since` here about this change?

Reply:
Nothing changed in any API.

It's just that we now actually DO what the doc block claimed. We always return a string. To my understanding that's no use-case for an `@since`

Reply:
The return type has not changed; the description just says the returned string can be empty. I would not add a `@since` tag either.